openmdao/core/problem.py

```diff
@@ -47,7 +66,7 @@ class Problem(System):
 
     """
 
-    def __init__(self, root=None, driver=None, impl=None):
+    def __init__(self, root=None, driver=None, impl=None, comm=None):
         super(Problem, self).__init__()
```

Comment:
Any tests where an alternative comm is added?

Also, what are the ramifications of this.